docs/source/development/debugging.md

```diff
@@ -1,83 +1,9 @@
-
-For guides on how to set up debugging with IDEs, please visit the [guide for debugging in VSCode](./set_up_vscode.md#debugging) and the [guide for debugging in PyCharm](./set_up_pycharm.md#debugging).
-
-## Debugging a node
-
-To start a debugging session when an uncaught error is raised within your `node`, implement the `on_node_error` [Hook specification](/api/kedro.framework.hooks):
-
-```python
-import pdb
-import sys
-import traceback
-
-from kedro.framework.hooks import hook_impl
-
-
-class PDBNodeDebugHook:
-    """A hook class for creating a post mortem debugging with the PDB debugger
-    whenever an error is triggered within a node. The local scope from when the
-    exception occured is available within this debugging session.
-    """
-
-    @hook_impl
-    def on_node_error(self):
-        _, _, traceback_object = sys.exc_info()
-
-        #  Print the traceback information for debugging ease
-        traceback.print_tb(traceback_object)
-
-        # Drop you into a post mortem debugging session
-        pdb.post_mortem(traceback_object)
-```
-
-You can then register this `PDBNodeDebugHook` in your project's `settings.py`:
-
-```python
-HOOKS = (PDBNodeDebugHook(),)
-```
-
-## Debugging a pipeline
-
-To start a debugging session when an uncaught error is raised within your `pipeline`, implement the `on_pipeline_error` [Hook specification](/api/kedro.framework.hooks):
-
-```python
-import pdb
-import sys
-import traceback
-
-from kedro.framework.hooks import hook_impl
-
-
-class PDBPipelineDebugHook:
-    """A hook class for creating a post mortem debugging with the PDB debugger
-    whenever an error is triggered within a pipeline. The local scope from when the
-    exception occured is available within this debugging session.
-    """
-
-    @hook_impl
-    def on_pipeline_error(self):
-        # We don't need the actual exception since it is within this stack frame
-        _, _, traceback_object = sys.exc_info()
-
-        #  Print the traceback information for debugging ease
-        traceback.print_tb(traceback_object)
-
-        # Drop you into a post mortem debugging session
-        pdb.post_mortem(traceback_object)
-```
-
-You can then register this `PDBPipelineDebugHook` in your project's `settings.py`:
-
-```python
-HOOKS = (PDBPipelineDebugHook(),)
-```
+* [Debugging in the CLI with pdb and Kedro Hooks](../hooks/common_use_cases.md#use-hooks-to-debug-your-pipeline)
+* [Debugging a Kedro project within a notebook](../notebooks_and_ipython/kedro_and_notebooks.md#debugging-a-kedro-project-within-a-notebook)
+* [Debugging in VSCode](./set_up_vscode.md#debugging)
+* [Debugging in PyCharm](./set_up_pycharm.md#debugging)
```

Comment:
I think the only problem I have with this is that some users (less experienced) may not know what they want. It could be that "if you have to ask, you don't want it" but for a new user, the title "Debugging in the CLI with pdb and Kedro Hooks" is going to be a bit intimidating. They're more likely to want one of the other options. 

Two things you could do:

1. Just reorder from easiest for newbies to understand and most likely they'll want. Probably notebook, VSCode, PyCharm and CLI.
2. Add a line of copy to each to say a bit more about what each page contains.